packages/ds/core/_index.scss

```diff
@@ -110,5 +112,7 @@
         @include focus.create();
     } @else if $key == disable {
         @include disable.create();
+    } @else if $key == checkbox-component {
```

Comment:
Not a fan of long stretch of `if else`. Can we refactor to https://stackoverflow.com/questions/16516263/does-sass-have-a-switch-function ?

Reply:
I'm somewhat confused here -  a switch should generally semantically be the same as a long series of elseif's. I'll have to check it in scss